examples/verif-manual/vm-299.py

```diff
@@ -0,0 +1,264 @@
+# GENERATE GEOMETRY
+H = 0.5
+mapdl.dim("A", "ARRAY", 3)
+mapdl.dim("B", "ARRAY", 3)
+mapdl.dim("C", "ARRAY", 3)
+
+mapdl.vfill("A(1)", "DATA", 0)
+mapdl.vfill("A(2)", "DATA", 2.0)
+mapdl.vfill("A(3)", "DATA", LX)
+mapdl.vfill("B(1)", "DATA", 0)
+mapdl.vfill("B(2)", "DATA", 2.0)
+mapdl.vfill("B(3)", "DATA", LY)
+mapdl.vfill("C(1)", "DATA", 0)
+mapdl.vfill("C(2)", "DATA", 2.0)
+mapdl.vfill("C(3)", "DATA", LZ)
+
+# Enter non-interactive mode
+with mapdl.non_interactive:
+    mapdl.run("*DO,I,1,2")
+    mapdl.run("*DO,J,1,2")
+    mapdl.run("*DO,K,1,2")
+    mapdl.block("A(I)", "A(I+1)", "B(J)", "B(J+1)", "C(K)", "C(K+1)")
+    mapdl.run("*ENDDO")
+    mapdl.run("*ENDDO")
+    mapdl.run("*ENDDO")
+# mapdl.aplot()
+
+# Generates new volumes by “gluing” volumes.
+mapdl.vglue("ALL")
+# Define element, 3-D Acoustic Fluid 20-Node Solid Element
+mapdl.et(1, 220, 3, 4)
+mapdl.type(1)  # set element type, Type=1
+mapdl.mat(1)  # set material type, MAT=1
+mapdl.esize(H)  # Specifies the element size.
+
+# Generates nodes and volume elements within volumes.
+mapdl.vmesh(9, 15, 1)
+
+# For elements that support multiple shapes, specifies the element shape, set mshape=3D
+mapdl.mshape(0, "3D")
+# Generates nodes and volume elements within volumes.
+mapdl.vmesh(1)
+# select nodes of specified location
+mapdl.nsel("S", "LOC", "X", 0)
+mapdl.nsel("A", "LOC", "X", LX)
+mapdl.nsel("A", "LOC", "Y", 0)
+mapdl.nsel("A", "LOC", "Y", LY)
+mapdl.nsel("A", "LOC", "Z", 0)
+mapdl.nsel("A", "LOC", "Z", LZ)
+
+# Define Absorption coefficient and transmission loss
+mapdl.sf("ALL", "ATTN", ALPHA)
+# Selects all entities
+mapdl.allsel()
+# select nodes of specified location
+mapdl.nsel("S", "LOC", "X", "A(2)")
+mapdl.nsel("R", "LOC", "Y", "B(2)")
+mapdl.nsel("R", "LOC", "Z", "C(2)")
+
+# Define Mass source; mass source rate; or power source
+# in an energy diffusion solution for room acoustics
+mapdl.bf("ALL", "MASS", WS)
+
+# Selects all entities
+mapdl.allsel()
+mapdl.eplot()
+# Finish pre-processing processor
+mapdl.finish()
+
+# Enter the solution processor to define solution controls
+mapdl.slashsolu()
+
+# Enter non-interactive mode
+with mapdl.non_interactive:
+    # redirects solver output to a file named "SCRATCH"
+    mapdl.run("/OUT,SCRATCH")
+    # SOLVE STATIC ANALYSIS
+    mapdl.solve()
+    # exists solution processor
+    mapdl.finish()
+
+    # Enter POST1 module (Post-processing processor)
+    mapdl.post1()
+    # Set the current results set to the last set to be read from result file
+    mapdl.set("LAST")
+    # Defines a path name and establishes parameters for the path
+    mapdl.path("X_SPL", 2, "", 15)
+    mapdl.ppath(1, "NODE", 0, 15, 1)
+    mapdl.ppath(2, "NODE", 30, 15, 1)
+    # Interpolates an item onto a path.
+    mapdl.pdef("UX", "U", "X", "NOAV")
+    mapdl.pdef("SPLX", "SPL", "", "NOAV")
+
+    # redirects output to the default system output file
+    mapdl.run("/OUT")
+    # reactivates suppressed printout
+    # mapdl.gopr()
```

Comment:
I think that it's in `mapdl.non_interactive` mode just for the printing. If it's not the case, please ignore my comment: 
```suggestion
# SOLVE STATIC ANALYSIS
mapdl.solve()
# exists solution processor
mapdl.finish()

# Enter POST1 module (Post-processing processor)
mapdl.post1()

with mapdl.non_interactive:
    # Defines a path name and establishes parameters for the path
    mapdl.path("X_SPL", 2, "", 15)
    mapdl.ppath(1, "NODE", 0, 15, 1)
    mapdl.ppath(2, "NODE", 30, 15, 1)
    # Interpolates an item onto a path.
    mapdl.pdef("UX", "U", "X", "NOAV")
    mapdl.pdef("SPLX", "SPL", "", "NOAV")
    mapdl.gopr()
```

Reply:
Done